go/stellar/build.go

```diff
@@ -18,6 +19,107 @@ import (
 	stellarAddress "github.com/stellar/go/address"
 )
 
+func ShowAdvancedSendBanner(mctx libkb.MetaContext, remoter remote.Remoter, from stellar1.AccountID, to string) (shouldShow bool, err error) {
+	// We don't show a banner if either the from or to are empty strings since they obviously don't have anything in
+	// common
+	if from == "" {
+		return false, nil
+	}
+	if to == "" {
+		return false, nil
+	}
+
+	// We show the advanced send form if there are any enabled choices for sending an asset
+	// Note that GetSendAssetChoicesLocal does not include native assets
+	res, err := GetSendAssetChoicesLocal(
+		mctx,
+		remoter,
+		stellar1.GetSendAssetChoicesLocalArg{
+			From: from,
+			To:   to,
+		})
+	if err != nil {
+		return false, err
+	}
+	for _, choice := range res {
+		if choice.Enabled {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+func GetSendAssetChoicesLocal(mctx libkb.MetaContext, remoter remote.Remoter, arg stellar1.GetSendAssetChoicesLocalArg) (res []stellar1.SendAssetChoiceLocal, err error) {
```

Comment:
Note that this method was moved from `frontend.go` so that I could reference it from within this file.